docs/release-notes/saros-e_15.0.0.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Saros Release Notes
+---
+
+The Saros Team is happy to announce Saros for Eclipse 15.0.0 and Saros for IntelliJ 0.1.0.
+
+Since the last release of Saros for Eclipse we mostly restructured, refactored and reworked the project in more than 1000 commits. From now on we want to use a lot shorter release cycles in order to
+provide fixes and features faster.
+
+Saros for IntelliJ 0.1.0 is the first alpha release version of our IntelliJ version. Please, see the [Saros for IntelliJ release notes](saros-i_0.1.0.md) before you use the new plug-in.
+
+## Upgrade Instruction
+We changed the structure of our configuration files and the location of our update-site.
+Therefore, if you used the previuous Saros version, please:
+* Remove your saros directory at `$HOME/.saros`
```

Comment:
Well I guess we will have to convert the account store as we did always when we changed things.

Reply:
Yes, this is the solution which is cleaner, but I am not sure whether it is worth the effort. I would expect that the benefit of the users is small. In any case, the users have an increased migration effort with this release. The old Saros plugin also has to be de-installed (we changed the names of the OSGI bundles) before the new version can be installed. Otherwise, the eclipse instance contains two installations.

Reply:
I'm Ok with not converting the config this time and as nobody volunteered at the last Stand-Up it's more or less up to @srossbach if he wants to implement this.

Nevertheless (on my system) if Saros can't read the config file it just rans into a silent exception and nothing else happens. So is it really needed to delete the file for Upgrading?
While locating the folder on Linux is easy, I think some Windows people will have to search for it.

Reply:
Ok, it is not necessary to delete the file. The user only sees that the account store is empty/no account is configured. If the user configured a new account, the invalid file is removed and the new account store is valid. Therefore no action is required, but I will add the information that the accounts have to be re-configured.